Sources/swift-format/Subcommands/LintFormatOptions.swift

```diff
@@ -49,6 +49,9 @@ struct LintFormatOptions: ParsableArguments {
     """)
   var ignoreUnparsableFiles: Bool = false
 
+  @Flag(help: "Whether or not to run formatting or linting in parallel consuming all possible resources")
```

Comment:
```suggestion
  /// Whether or not to run the formatter/linter in parallel.
  @Flag(help: "Process files in parallel, simultaneously across multiple cores.")
```

Reply:
Suggested edit to match existing documentation and help text convention.

As far as the note about "consuming all possible resources", I wasn't able to find any definitive, on-the-record language from Apple to support that. The most they've committed to publicly (that I can find, at least) was from [WWDC 2017 Session 706](https://asciiwwdc.com/2017/sessions/706?q=parallel).